lib/command/run_logs.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Command
+  class RunLogs < Base
```

Comment:
The code here is essentially the same that was in `run:detached` for fetching the logs, with minor modifications, like getting the exit code from the command.